src/kepler_model/server/model_server.py

```diff
@@ -189,6 +189,7 @@ def get_available_models():
     fg = request.args.get("fg")
     ot = request.args.get("ot")
     energy_source = request.args.get("source")
+    node_type = request.args.get("type")
```

Comment:
Just a thought ... `type` is too generic.  should we call this node-type itself ?